commit 24b937c5b96688bafbd3c4a44cd03e166bab8f2f
Author: jinwei <jinwei@sensetime.com>
Date:   Thu May 28 15:40:05 2015 +0800

    tab 8 for company

diff --git a/emacs.conf/.emacs b/emacs.conf/.emacs
index 1a37842..94154d1 100644
--- a/emacs.conf/.emacs
+++ b/emacs.conf/.emacs
@@ -222,9 +222,9 @@
 ; C/C++
 ;--------------------------------------------------------------------
 (setq c-default-style "BSD")
-(setq c-basic-offset 4)
-(setq c-tab-width 4)
-(setq c-indent-level 4)
+(setq c-basic-offset 8)
+(setq c-tab-width 8)
+(setq c-indent-level 8)
 
 
 ;shell,gdb退出后，自动关闭该buffer
